### PR TITLE
Updated steps for get scenario

### PR DESCRIPTION
**Pull request by cfnyu**
Fixes for behave

### CODE REVIEW

**Review by sfloam — APPROVED** [review]
Thanks for fixing this.

File: features/steps/recommendations_steps.py [hunk 1]
```diff
@@ -70,11 +70,11 @@ def step_impl(context):
     assert found.text != 'batman'
     context.driver.save_screenshot('line67.png')
 
-@then(u'I will see a "product_id" with "45" in my results')
+@then(u'I will see a "product_id" with "29" in my results')
```
Comment (Harivadhana) [inline features/steps/recommendations_steps.py:73]: I think we can change this hard-coded values to the already existing step function of list or query scenarios

**Comment by cfnyu** [comment]
IBM Cloud [toolchain](https://console.bluemix.net/devops/toolchains/b70ebc13-f2da-447f-9876-2a7d703481b6 "toolchain-nyu-recommendation-service-f17"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/f3063f54-6e1d-4e45-8b7c-b27afa21a694 "delivery") deployed [recommendations](https://github.com/devopsdelta/recommendations/releases/tag/deploy-ibm-yp-us-south-nyu-delta-team-f17-dev-20171213-225016 "deploy-ibm-yp-us-south-nyu-delta-team-f17-dev-20171213-225016") to [dev](https://console.bluemix.net/apps/c7225f97-1211-46e5-9fbc-8559da2c5fe7?env_id=ibm:yp:us-south "ibm:yp:us-south:nyu-delta-team-f17:dev"), including commit edf2e3b139125bdba0d3a480c31a55430dde4f28

**Comment by cfnyu** [comment]
IBM Cloud [toolchain](https://console.bluemix.net/devops/toolchains/b70ebc13-f2da-447f-9876-2a7d703481b6 "toolchain-nyu-recommendation-service-f17"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/f3063f54-6e1d-4e45-8b7c-b27afa21a694 "delivery") deployed [recommendations](https://github.com/devopsdelta/recommendations/releases/tag/deploy-ibm-yp-us-south-nyu-delta-team-f17-prod-20171213-225345 "deploy-ibm-yp-us-south-nyu-delta-team-f17-prod-20171213-225345") to [prod](https://console.bluemix.net/apps/f9460b21-5cc6-4470-b00b-9de89c5a370a?env_id=ibm:yp:us-south "ibm:yp:us-south:nyu-delta-team-f17:prod"), including commit edf2e3b139125bdba0d3a480c31a55430dde4f28

**Pull request merged**